.yarnrc.yml

```diff
@@ -6,6 +6,8 @@ nmHoistingLimits: workspaces
 
 nmSelfReferences: false
 
+checksumBehavior: update
```

Comment:
Had to add this to fix the cached package issue in CI.

`The remote archive doesn't match the expected checksum`

Reply:
I guess we can remove it in another pr once the issue above is fixed in ci.